valhalla/meili/state.h

```diff
@@ -43,12 +43,17 @@ class State {
     // Cache results
     label_idx_.clear();
     uint16_t dest = 1; // dest at 0 is reserved for the origin
+                       //
+#ifdef LOGGING_LEVEL_TRACE
     uint16_t found = 0;
```

Comment:
instead of adding the `ifdef`s let's do:
```suggestion
    [[maybe_unused]] uint16_t found = 0;
```